euphonic/cli/utils.py

```diff
@@ -934,3 +934,9 @@ def _compose_style(
 
     style.append(explicit_args)
     return style
+
+def _get_title(filename: str, title: str = '') -> str:
+    """Get a plot title: either user-provided string, or from filename"""
+    if title:
+        return title
+    return pathlib.Path(filename).stem
```

Comment:
Warrant using a ternary if? 

Would it be better to use `None`? Is there a reason a user might want to have an empty title?

Reply:
Ok, I would say that's a different issue and if this is the behaviour elsewhere that's fine. May want to make an issue even if it just sits there.

Reply:
The fix required fewer changes than I expected, let's give it a try